### DISCUSSION TITLE
concourse 7.4 upgrade software compatibility

### DISCUSSION THREAD

**Discussion by saketlj** [concourse/concourse#7811] [Help & Support]
Hi,

I am upgrading concourse to 7.4 version and I was looking for any documentation available with software compatibility matrix for supporting softwares like bpm and postgres. I did went through release-notes here https://github.com/concourse/concourse/releases/tag/v7.4.0 but didn't find this information.

Currently we are on concourse 6.7 and below are supporting software and there versions we are using.
- bpm/1.1.9
- postgres/43
- concourse/6.7.4

**Comment by crsimmons** [comment]
Concourse itself is intentionally agnostic to how it is deployed so the release notes don't contain details like that.

The [concourse-bosh-deployment repo](https://github.com/concourse/concourse-bosh-deployment) is community maintained but is probably the best source for bosh deployment questions. You can see the current suggested versions of the various releases in [versions.yml](https://github.com/concourse/concourse-bosh-deployment/blob/master/versions.yml) from that repo. There isn't a compatibility matrix afaik but you might be able the piece some of it together from the git history of that file.
